odl/discr/discretization.py

```diff
@@ -543,6 +543,13 @@ def __init__(self, space, data):
         assert isinstance(space, DiscretizedSpace)
         DiscretizedSetVector.__init__(self, space, data)
 
+    def __ipow__(self, p):
+        """Implement ``self **= p``."""
+        # Falls back to `LinearSpaceVector.__ipow__` if `self.ntuple`
+        # has no own `__ipow__`. The fallback only works for integer `p`.
```

Comment:
It seems you are correct, needed some thought here :/

Anyway:

> Frankly we should make passthroughs for all the arithmetic things in case someone overrides in the ntuple.